### PR TITLE
Create test.md

### PR DESCRIPTION
**Pull request by adityav-verma**
## What?
List the changes being proposed.

## Why?
Explain why / also link the issue that is solved by this (if applicable).

## Config changes:
```
key: value
```

## Testing Scope:
- [ ] Basic sanity
- [ ] Migration
- [ ] Edge cases

## Dependent Pull Requests:
Link all the dependent Pull Requests.

### CODE REVIEW

File: test.md [hunk 1]
```diff
@@ -0,0 +1 @@
+
```
Comment (adityav-verma) [inline test.md:3]:
```suggestion
## What?
Add what this file is about?

## Why?
Add why this file is required?
```
